e3db/src/main/java/com/tozny/e3db/Client.java

```diff
@@ -173,8 +172,120 @@ public void execute(Runnable runnable) {
 
     mapper = new ObjectMapper();
     mapper.configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true);
+
+    if (Platform.isAndroid()) {
+      anonymousClient = enableTLSv12(new OkHttpClient.Builder()).build();
+    } else {
+      anonymousClient = new OkHttpClient.Builder().build();
```

Comment:
I wonder if this is worth doing for plain java as well. Otherwise, is there a way to detect if TLSv12 will be used/is available? Would be nice to fail early if it wasn't.